kuma/static/styles/minimalist/print.scss

```diff
@@ -0,0 +1,69 @@
+@import '../includes/vars';
+@import './mixins/print-mixins';
+
+@media print {
```

Comment:
What's the point of this? If this file is in a `<link href=print.css media=print>` it is already for printing.

Reply:
Very true, there is no need for it here anymore.